warp10/src/main/java/io/warp10/plugins/influxdb/InfluxDBWarp10Plugin.java

```diff
@@ -94,18 +96,19 @@ public void init(Properties properties) {
     this.port = Integer.parseInt(properties.getProperty(CONF_INFLUXDB_PORT, "8086"));
     this.host = properties.getProperty(CONF_INFLUXDB_HOST, "127.0.0.1");
     this.token = properties.getProperty(CONF_INFLUXDB_DEFAULT_TOKEN);
-    
+    this.measurementlabel = properties.getProperty(CONF_INFLUXDB_MEASUREMENT_LABEL);
+
```

Comment:
Why don't you hardcode this feature ?
(users and support team will recognize what was imported easily)

Reply:
Because it may interfer with a name that is already in use.